app/static/Climate-Mind_bundled.yml

```diff
@@ -75,6 +46,36 @@ paths:
                         createdByUserId: 39229453-E000-4801-91CA-D1F85E750508
                         createdDateTime: 'Wed, 05 Jan 2022 06:44:33 GMT'
                         invitedUserName: Jim 4
+                properties:
```

Comment:
The examples used in the documentation should match these changes shown here too. I believe the examples shown here don’t match with your changes. Can you make them match?

Reply:
@cyplas I know you've asked to change docs for the endpoint but I worked on PUT endpoint instead and didn't touch the GET endpoint to which this example belongs to. 
Since you will be around next week, could you please change this yourself? 
Thanks!

Reply:
I took a look, but I think there's a little confusion with which route we are talking about: @danmash slipped in an "s" in the pull request title, and @rodriguesk referenced the /conversations route, which we're not working on here. For /conversation, I double checked again and only found a mistake in conversationStatus for one of the examples, which is deprecated anyway.  I tweaked a couple more things, but quite minor.